base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
What's canonical about WINNT? uname is a posix thing. The Julia function names and the Julia constant identifiers should be consistent, I don't see why shell scripting jargon matters.